packages/ra-ui-materialui/src/list/ListView.tsx

```diff
@@ -68,10 +68,15 @@ export const ListView = (props: ListViewProps) => {
                     })}
                     key={version}
                 >
-                    {bulkActionButtons !== false && bulkActionButtons && (
-                        <BulkActionsToolbar {...controllerProps}>
-                            {bulkActionButtons}
-                        </BulkActionsToolbar>
+                    {bulkActionButtons === true ? (
+                        <DefaultBulkActionButtons />
+                    ) : (
```

Comment:
I believe `DefaultBulkActionButtons` should be passed as the `BulkActionsToolbar` child